jupyter/README.md

```diff
@@ -22,35 +22,12 @@
 
 Swan uses *Jupyter Notebook* to filter, process and visualize results from experiments.
 
-## Installation
-
-In order to install Jupyter, you need `python` and `pip` installed. On Centos 7, installation of those packages can be achieved by:
+You need to have Docker installed to run Jupyter notebooks easily. We have provided a Docker image that will make running notebooks as easy as possible. You just need to run:
 
 ```sh
-sudo yum install python-pip python-devel
+sudo docker run --net=host intelsdi/swan-jupyter
```

Comment:
even if `--net host` effect is the same it is better to expose port 8888 explicitly to listen only on localhost

so `--net host`  should be replace with 

```
-p 127.0.0.1:8888:8888
```
for security reasons - - we shouldn't assume that base jupyter image will not change this sane default

```
-p=[]      : Publish a container᾿s port or a range of ports to the host
               format: ip:hostPort:containerPort | ip::containerPort | hostPort:containerPort | containerPort
               Both hostPort and containerPort can be specified as a
               range of ports. When specifying ranges for both, the
               number of container ports in the range must match the
               number of host ports in the range, for example:
                   -p 1234-1236:1234-1236/tcp
```
https://docs.docker.com/v1.11/engine/reference/run/